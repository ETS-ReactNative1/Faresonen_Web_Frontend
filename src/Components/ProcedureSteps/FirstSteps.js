import React from "react";
import { Icon, Step } from "semantic-ui-react";

function FirstSteps() {
	return (
		<Step.Group>
			<Step active>
				<Icon name="soccer" />
				<Step.Content>
					<Step.Title>Velg liga</Step.Title>
					<Step.Description>
						Hvilken liga skal du generere lefse for?
					</Step.Description>
				</Step.Content>
			</Step>

			<Step disabled>
				<Icon name="numbered list" />
				<Step.Content>
					<Step.Title>Velg runde</Step.Title>
					<Step.Description>
						Hvilken runde gjelder lefsen for?
					</Step.Description>
				</Step.Content>
			</Step>

			<Step disabled>
				<Icon name="print" />
				<Step.Content>
					<Step.Title>Print</Step.Title>
					<Step.Description>
						Ctrl + P for Ã¥ printe lefsen
					</Step.Description>
				</Step.Content>
			</Step>

			<Step disabled>
				<Icon name="cocktail" />
				<Step.Content>
					<Step.Title>Bra jobbet</Step.Title>
					<Step.Description>
						Ta deg en velfortjent pause!
					</Step.Description>
				</Step.Content>
			</Step>
		</Step.Group>
	);
}

export default FirstSteps;
