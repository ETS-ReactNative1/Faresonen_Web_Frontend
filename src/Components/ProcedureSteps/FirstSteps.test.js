import React from "react";
import renderer from "react-test-renderer";
import FirstSteps from "./FirstSteps";

describe("<FirstSteps />", () => {
  it("Should render properly", () => {
    const tree = renderer.create(<FirstSteps />);
    expect(tree).toMatchSnapshot();
  });
});
