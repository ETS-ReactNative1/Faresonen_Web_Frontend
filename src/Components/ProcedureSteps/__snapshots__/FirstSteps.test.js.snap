// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FirstSteps /> Should render properly 1`] = `
<div
  className="ui steps"
>
  <div
    className="active step"
    href={undefined}
    onClick={[Function]}
  >
    <i
      aria-hidden="true"
      className="soccer icon"
    />
    <div
      className="content"
    >
      <div
        className="title"
      >
        Velg liga
      </div>
      <div
        className="description"
      >
        Hvilken liga skal du generere lefse for?
      </div>
    </div>
  </div>
  <div
    className="disabled step"
    href={undefined}
    onClick={[Function]}
  >
    <i
      aria-hidden="true"
      className="numbered list icon"
    />
    <div
      className="content"
    >
      <div
        className="title"
      >
        Velg runde
      </div>
      <div
        className="description"
      >
        Hvilken runde gjelder lefsen for?
      </div>
    </div>
  </div>
  <div
    className="disabled step"
    href={undefined}
    onClick={[Function]}
  >
    <i
      aria-hidden="true"
      className="print icon"
    />
    <div
      className="content"
    >
      <div
        className="title"
      >
        Print
      </div>
      <div
        className="description"
      >
        Ctrl + P for Ã¥ printe lefsen
      </div>
    </div>
  </div>
  <div
    className="disabled step"
    href={undefined}
    onClick={[Function]}
  >
    <i
      aria-hidden="true"
      className="cocktail icon"
    />
    <div
      className="content"
    >
      <div
        className="title"
      >
        Bra jobbet
      </div>
      <div
        className="description"
      >
        Ta deg en velfortjent pause!
      </div>
    </div>
  </div>
</div>
`;
